cql2pgjson/src/test/java/org/z3950/zing/cql/cql2pgjson/IndexPerformanceTest.java

```diff
@@ -114,7 +114,7 @@ public void cqlValue() throws CQL2PgJSONException, IOException {
   public void cqlNum() throws CQL2PgJSONException, IOException {
     runSqlStatement("DROP INDEX IF EXISTS idx_num;");
     runSqlStatement("CREATE INDEX idx_num ON config_data ((jsonb->'num'));");
-    CQL2PgJSON cql2pgJsonSchema = new CQL2PgJSON("jsonb", Util.getResource("indexPerformanceTest.json"));
+    CQL2PgJSON cql2pgJsonSchema = new CQL2PgJSON("jsonb");
```

Comment:
rename all subsequent uses of cql2pgJsonSchema to cql2pgJson and remove this line:

    CQL2PgJSON cql2pgJsonSchema = new CQL2PgJSON("jsonb");